### PR TITLE
[#103] Manage External Dependencies

### PR DESCRIPTION
**Pull request by ruspl-afed**
Remove unnecessary maven egit repo

### CODE REVIEW

File: org.eclipse.mylyn-target/mylyn-e4.26.target
Comment (BeckerFrank) [inline org.eclipse.mylyn-target/mylyn-e4.26.target:126]: Where do wie then find the org.eclipse.egit.github.core plugin which is used by org.eclipse.mylyn.github.core?

Comment (ruspl-afed) [reply org.eclipse.mylyn-target/mylyn-e4.26.target:126]: My bad, fixed

Comment (BeckerFrank) [reply org.eclipse.mylyn-target/mylyn-e4.26.target:126]: Will post soon an update after I have tried that https://download.eclipse.org/egit/github/updates as p2 works.

Comment (ruspl-afed) [reply org.eclipse.mylyn-target/mylyn-e4.26.target:126]: Please reference the exact bundle as IU if you want to switch there

Comment (ruspl-afed) [reply org.eclipse.mylyn-target/mylyn-e4.26.target:126]: And then please merge this one @BeckerFrank if it works for you

Comment (BeckerFrank) [reply org.eclipse.mylyn-target/mylyn-e4.26.target:126]: > Will post soon an update after I have tried that https://download.eclipse.org/egit/github/updates as p2 works.

Sorry this did not work because the plugin org.eclipse.egit.github.core_6.1.0.202203080745-r.jar is not installable with  a separate feature. It is included in the feature org.eclipse.mylyn.github.feature_6.1.0.202203080745-r.jar

We need to request that org.eclipse.egit.github.core should be a standalone feature.

If we need to change the target file and remove the maven location I think we should try.
download the needed jar into a folder parallel to the target file and use Type Directory instead of Maven.
But for this case, I can't find a value for the path that works for a maven build and for the running eclipse installation.

**Comment by wimjongman** [comment]
Hey let me know when to test. I was away for a conference but back again. Just tried to install gh connector but it fails trying to uninstall egit.

**Comment by BeckerFrank** [comment]
@wimjongman , @ruspl-afed:
I think we can merge this!
I was the last who changed the file so maybe someone else can do the review and merge.

Actual I do a combined test for #104 and #105 on my local env to verify that the update site and installation are ok.

**Review by wimjongman — APPROVED** [review]
LGTM.

**Comment by ruspl-afed** [comment]
I cannot add a review/approval since I was a creator.

BTW, how about to allow Project Leads merge without required approval? It may be required for some cases.

**Pull request merged**
